acl/src/test/java/org/apache/rocketmq/acl/plain/PlainPermissionManagerTest.java

```diff
@@ -45,6 +54,8 @@ public class PlainPermissionManagerTest {
     PlainAccessConfig plainAccessConfig = new PlainAccessConfig();
     Set<Integer> adminCode = new HashSet<>();
 
+    private static final String PATH = "src/test/resources";
```

Comment:
Why not load test resources from classpath? It is a much more idiomatic

Reply:
I also think so!It is a much more idiomatic!

Reply:
Thanks  for your review, I have  solved it